che/service/che_service.go

```diff
@@ -76,7 +77,12 @@ func (s *cheServiceImpl) DeleteUser(ctx context.Context, identity repository.Ide
 			"response_status": res.Status,
 			"response_body":   bodyString,
 		}, "unable to delete user in Che")
-		return errs.Wrapf(err, "unable to delete user '%s' in Che", identity.ID.String())
+		if res.StatusCode == http.StatusNotFound {
```

Comment:
should we log with a `WARN` level if we get a `404`, and log with an `ERROR` level otherwise?

Reply:
also, you could probably use a `switch(res.StatusCode) {...}` block here to treat the different kind of responses (204/404/others)

Reply:
Done in https://github.com/fabric8-services/fabric8-auth/pull/836/commits/611f09ae78bbdf18ce1d58dce357e9f203e1eaee